python/valis/db/queries.py

```diff
@@ -264,6 +264,62 @@ def carton_program_search(name: str, name_type: str) -> peewee.ModelSelect:
     )
 
 
+def get_targets_obs(release: str, obs: str, obsWave: str) -> peewee.ModelSelect:
+    """ Return all targets with spectra from a given observatory
+
+    Parameters
+    ----------
+    release : str
+        the data release to look up
+
+    obs: str
+        Observatory to get targets from. Either 'APO' or 'LCO'
+
+    obsWave: str
+        Which spectrograph to return data from. Can be 'boss',
+        'apogee' or 'all' for both.
```

Comment:
should this parameter be renamed `spectrograph` or `instrument` to make it more explicit?

Reply:
This makes sense. I have switched this to `spectrograph`.